<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MorphingViewer</class>
 <widget class="QWidget" name="MorphingViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Morphing</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAddMesh">
       <property name="text">
        <string>Add Mesh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSaveMesh">
       <property name="text">
        <string>Save Mesh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="editMorphingWeights"/>
     </item>
     <item>
      <widget class="QSlider" name="sliderMorphing">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QVTKWidget" name="qvtkWidgetMorphing"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
